states/georgia/elections.toml

```diff
@@ -3,17 +3,79 @@ date = 2020-06-09
 name = "Presidential Primary"
 original_date = 2020-06-09
 
-[20200811]
-date = 2020-08-11
-name = "Congressional Primary Run-off (if needed)"
-original_date = 2020-08-11
+# [20200811]
+# date = 2020-08-11
+# name = "Congressional Primary Run-off (if needed)"
+# original_date = 2020-08-11
+# There is no run-off this election cycle
```

Comment:
It looks like it is happening: https://sos.ga.gov/index.php/elections/last_day_to_register_for_federal_and_state_runoff_elections Do you want to add it?

Reply:
added the runoff